### PR TITLE
@nico/choppy scroll over discover pools

### PR DESCRIPTION
**Pull request by nicoglennon**
Fix choppy scrolling over pools. Only noticeable in iOS device (you can't see it in sim). Fixes RNBW-1443.

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
lgtm 420 blaze it

**Review by mikedemarais — APPROVED** [review]
dank thx

File: src/hooks/useScanner.js [hunk 1]
```diff
@@ -194,10 +194,11 @@ export default function useScanner(enabled) {
     [
       isScanningEnabled,
       disableScanning,
-      handleScanAddress,
       handleScanWalletConnect,
-      handleScanRainbowProfile,
       handleScanInvalid,
+      handleScanEthereumUrl,
+      handleScanAddress,
```
Comment (mikedemarais) [inline src/hooks/useScanner.js:200]: abc

**Review by mikedemarais — APPROVED** [review]
i left an `abc` but then yolo send it

**Pull request merged**